interfaces/builtin/basedeclaration.go

```diff
@@ -204,6 +204,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  chroot:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
This can be auto-connected. The interface grants precisely what it says it does and it does not provide privileged access to the system. There's an argument to be made for adding it to the default policy, but I think having it in its own interface is ok-- most snaps won't need it.